src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -290,6 +287,11 @@ public void getTriggers(@NonNull Callback<List<Trigger>> callback) {
         getPipe(pipeName).read(getFilter(uri), callback);
     }
 
+    @SuppressWarnings("unchecked")
+    private <T> void savePipe(String pipeName, Object object, Callback<List<T>> callback, Class pipeClass) {
+        getPipe(pipeName).save(pipeClass.cast(object), callback);
```

Comment:
@danielpassos Thanks. I didn't checked. Just thought it will be needing to know from which class object relate to. But checked its not. :)